kyo-combinators/shared/src/test/scala/kyo/ResourceCombinatorTest.scala

```diff
@@ -17,14 +17,13 @@ class ResourceCombinatorTest extends Test:
                     _      <- setter(50)
                     result <- IO(state)
                 yield result
-            val beforeResources               = scala.concurrent.Future(assert(state == 0))
```

Comment:
@johnhungerford I'm not sure what the goal was here but, given that `Future` will fork the computation, there isn't a guarantee that it will run before `Resource.run`.